### ISSUE TITLE
yarn docker setup not working on macOS

### ISSUE THREAD

**Issue by aust** [graphile/starter#200]
### Summary

When running `yarn docker setup` on macOS, it fails with error:

```
? What would you like to call your database? graphile_starter
Installing or reinstalling the roles and database...
error: password authentication failed for user "postgres"
    at Connection.parseE (/work/node_modules/pg/lib/connection.js:581:48)
    at Connection.parseMessage (/work/node_modules/pg/lib/connection.js:380:19)
    at Socket.<anonymous> (/work/node_modules/pg/lib/connection.js:116:22)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    at readableAddChunk (_stream_readable.js:271:9)
    at Socket.Readable.push (_stream_readable.js:212:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {
  length: 104,
  severity: 'FATAL',
  code: '28P01',
  detail: undefined,
  hint: undefined,
  position: undefined,
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'auth.c',
  line: '337',
  routine: 'auth_failed'
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process exited with status '1' (running 'yarn compose run server yarn setup starter')
    at runSync (/Users/austin/Development/starter/scripts/lib/run.js:31:13)
    at main (/Users/austin/Development/starter/docker/scripts/yarn-setup.js:76:3)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### Steps to reproduce

1. Clone the repo to directory `starter`
2. `cd starter`
3. `export UID; yarn docker setup`

### Expected results

Setup completes successfully.

### Actual results

Setup fails with error `password authentication failed for user "postgres"`.

### Additional context

`starter_db_1` container is successfully created. Attempting to connect using a local psql command `psql "postgres://postgres:<PASSWORD>@localhost:6543/postgres"` also fails with the same error.

I've tested this and it fails even after wiping out the `starter` directory and re-cloning.

I've also wiped out my docker containers and started fresh, but I still receive this error.

### Possible Solution

It looks like there's an issue with how the Postgres db credentials are configured.

**Comment by benjie** [comment]
@aust Not sure what's going on there; when you run `yarn docker setup` a file `docker/.env` should be created which should contain a line starting `POSTGRES_PASSWORD=`. Using that password you should be able to connect to the database - does that work?

My suspicion was an old `.env` file in the root from a local (non-Docker) setup, but you've confirmed you've wiped the directory so it can't be that.

**Comment by benjie** [comment]
Actually this seems to be something #196 broke; I've fixed it in #165 which I'll be merging shortly
